src/control/cmd/daos/pretty/pool.go

```diff
@@ -98,3 +98,187 @@ func PrintPoolQueryTargetInfo(pqti *daos.PoolQueryTargetInfo, out io.Writer) err
 
 	return w.Err
 }
+
+func poolListCreateRow(pool *daos.PoolInfo, upgrade bool) txtfmt.TableRow {
```

Comment:
The `dmg` command code pulls in protobuf and other dependencies, and its pretty-printers work with types from `lib/control` and `lib/daos`. These pretty-printers work only with types from `lib/daos`. The goal I'm pushing toward is for `lib/daos` to have no dependencies on `lib/control` or the rest of the control plane code, because it should be possible to use the client API without pulling in all of that other stuff. The code isn't there yet, but that's where I'm trying to go with it.

Reply:
makes sense, thanks